dozer/cogs/general.py

```diff
@@ -1,10 +1,12 @@
 """General, basic commands that are common for Discord bots"""
-
+import asyncio
 import inspect
 import discord
 from discord.ext.commands import BadArgument, cooldown, BucketType, Group, has_permissions, NotOwner
 
+from . import _utils
```

Comment:
The `from ._utils import *` statement below this imports the names you need already. This line can be deleted and line 232 can use `DynamicPrefixEntry` directly instead of `_utils.DynamicPrefixEntry`.